compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -8304,6 +8304,8 @@ inlineP256Mod(TR::Node * node, TR::CodeGenerator * cg)
    return NULL;
    }
 
+// TODO: Merge the inlineReverseBytes methods into their respective evaluators once OpenJ9 stops referencing them
+
```

Comment:
Looks like Power has these same methods in OpenJ9. We should clean all of them up once this is merged.